src/run-tests/update-dotenv.py

```diff
@@ -0,0 +1,17 @@
+import dotenv
+import os
+import sys
+
+"""
+This file accepts 2 command line arguments:
+1. The key of the value we are overwriting in the dotenv file (i.e. ORDERS_API_URL)
+2. The new value of that key
+"""
+key = sys.argv[1]
+new_value = sys.argv[2]
+
+dotenv_file = dotenv.find_dotenv()
+dotenv.load_dotenv(dotenv_file)
+
+os.environ[key] = new_value
+dotenv.set_key(dotenv_file, key, os.environ[key])
```

Comment:
I am not sure what this file is for so I didn't use it. (Something around creating a dotenv file?)

If you think that this is cleaner than passing env vars, feel free to change the script.

Reply:
I created this to make sure the variables were synced up across all environments, but I can no longer see a need so I've deleted this file.